src/ui.form-switch/utilVariant.ts

```diff
@@ -0,0 +1,31 @@
+import { cva } from "../utils/ui.ts";
+import type { Config } from "./types.ts";
+
+export default class VariantService {
+  static get(config: Config) {
+    const { wrapper, circle, toggleLabel } = config;
+
+    const wrapperVariants = cva({
+      base: wrapper?.base,
+      variants: wrapper?.variants,
+      compoundVariants: wrapper?.compoundVariants,
+    });
+
+    const circleVariants = cva({
+      base: circle?.base,
+      variants: circle?.variants,
+      compoundVariants: circle?.compoundVariants,
+    });
+
+    const toggleLabelVariants = cva({
+      base: toggleLabel?.base,
+      compoundVariants: toggleLabel?.compoundVariants,
+    });
+
+    return {
+      wrapperVariants,
```

Comment:
need to discuss. It looks like a legacy code.

Reply:
Removed this file after discussion.